### PR TITLE
Initial common model checkin

### PR DESCRIPTION
**Pull request by kreinold**
Updated draft checkin from yesterday to address the following:

- Simplified several of the data types (“definitions”) for MVP, generally changing CodeableConcept to string.
- Extended the original CCDH model to add key search fields for CDA, including a harmonized_disease_type which in future versions will be calculated either during ETL or in the API implementation code. 
- Tried to follow CCDH conventions for naming fields and provided descriptions for additions.  Also for simplifications, I left notes regarding how it should work in future.
- Also added examples json files based on real data.
- Modified .gitignore to get rid of .DS_Store

### CODE REVIEW

**Comment by druth314** [comment]
Everything is looking great so far. The examples definitely help clear things up. As far as making things available in a BQ table go it may be difficult to implement the codeable concepts unless we make them records(nested structures)

**Review by druth314 — APPROVED** [review]

**Comment by pshapiro4broad** [comment]
Please remove the `.DS_Store` file from this PR, it's a mac specific metadata file that shouldn't be committed to git. You can add it to the repo's `.gitignore` to avoid having remember to not commit it.

File: src/schema/json/ResearchSubject.json [hunk 1]
```diff
@@ -1,40 +1,75 @@
...
+             },
+            "harmonized_disease_type": {
+                "description": "Harmonized disease type based on CCDH model; for MVP, the ETL process will assign the value of GDC disease_type.  This may be removed from the model in future and replaced with an API call that determines this value based submitting the CodeableConcept to a vocabulary service which will return the canonical value to be used for search.  Searching for any of the terms in the CodeableConcept is an option to be considered in future.  This field will not appear in data submitted by the DCs.",
+                "type": "string"
+            },
+            "days_to_birth": {
+                "description": "Per GDC Dictionary, number of days between the date used for index and the date from a person's date of birth represented as a calculated negative number of days.",
+                "type": "integer"            
+            },
+            "sex": {
```
Comment (pshapiro4broad) [inline src/schema/json/ResearchSubject.json:58]: Will these fields be harmonized, so a search for sex (or ethnicity or race) can be done across multiple datasets?

In a harmonized form I would expect this to map to an enumerated type that has all the possible legal values for this field.

File: src/schema/json/ResearchSubject.json [hunk 2]
```diff
@@ -1,40 +1,75 @@
...
+            },
+            "primary_disease_type": {
+                "$ref": "#/definitions/CodeableConcept",
+                "description": "The text term used to describe the type of malignant disease, as categorized by the World Health Organization's (WHO) International Classification of Diseases for Oncology (ICD-O). "
+            },
+            "taxon": {
+                "type": "string",
+                "description": "The taxonomic group (e.g. species) of the patient. For MVP, since taxonomy vocabulary is consistent between GDC and PDC, using text.  Ultimately, this will be a vocabulary service term."
+             },
+            "harmonized_disease_type": {
```
Comment (pshapiro4broad) [inline src/schema/json/ResearchSubject.json:50]: Can the schema specify all the possible harmonized disease types? That would help if we wanted to validate that data is correctly harmonized.

File: src/schema/json/ResearchSubject.json (hunk 1 above)
Comment (kreinold) [reply src/schema/json/ResearchSubject.json:58]: Need to discuss how to do this while following CCDH model and keeping BQ schema in mind.  I'll write up a separate ticket for it.

File: src/schema/json/ResearchSubject.json (hunk 2 above)
Comment (kreinold) [reply src/schema/json/ResearchSubject.json:50]: Added comment about this but generally the plan is to validate against the vocabulary service which is not yet available.  I'll write up another ticket to consider whether we should re-examine this plan.

**Review by kreinold — COMMENTED** [review]
All ok.

**Pull request merged**